templates/python/.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+---
+repos:
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ["--settings", "backend/setup.cfg"]
```

Comment:
I think you can remove the `backend/` prefix here (and everywhere else). This is only needed for projects where we have a backend and frontend in different subdirectories

Reply:
Ah, of course. done